netket/_src/vqs/transformed_vstate.py

```diff
@@ -25,11 +33,12 @@ def apply_operator(operator, vstate, *, seed=None):
     transformed_apply_fun, new_variables = make_logpsi_op_afun(
         vstate._apply_fun, operator, vstate.variables
     )
-    chunk_size = (
-        None
-        if vstate.chunk_size is None
-        else vstate.chunk_size / operator.max_conn_size
-    )
+
+    if vstate.chunk_size is None:
+        chunk_size = None
+
+    else:
+        chunk_size = vstate.chunk_size // operator.max_conn_size
```

Comment:
just to make sure it does not end up being 0...
```suggestion
        chunk_size = max(vstate.chunk_size // operator.max_conn_size, 1)
```